lib/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -1011,6 +1011,45 @@ struct ExpOpConversionApprox
   }
 };
 
+struct AbsIOpConversion
```

Comment:
Can we rely on `MathToLLVM` for that?

Reply:
I tried adding the MathToLLVM conversion patterns but it was rewriting the `tensor` version before the `ElementwiseOpConversion` had a chance to convert it to scalar.